modules/nw-ovn-kubernetes-live-migration-about.adoc

```diff
@@ -65,7 +65,7 @@ Before using the limited live migration method to the OVN-Kubernetes network plu
 
 * Egress router pods block the limited live migration process. They must be removed before beginning the limited live migration process.
 
-* During the limited live migration, multicast, egress IP addresses, and egress firewalls are temporarily disabled. They can be migrated from OpenShift SDN to OVN-Kubernetes after the limited live migration process has finished.
+* During the migration, when the cluster is running with both OVN-Kubernetes and OpenShift SDN, multicast and egress IP are temporarily disabled for both CNIs. Egress firewalls remains functional.
```

Comment:
This is likely my ignorance, but I read _multicast and egress IP_ as _multicast IP and egress IP_, Would a typical user make that mistake? 
Similarly, in the original it uses _egress IP **addresses**_; does the rewrite need _addresses_ to be clear?

Reply:
fixed. either would've been ok IMO